cmd/keytransparency-client/cmd/hammer.go

```diff
@@ -75,8 +73,23 @@ var hammerCmd = &cobra.Command{
 			return err
 		}
 
-		h.Run(ctx, maxOperations, maxWorkers, ramp)
-		return nil
+		return h.Run(ctx, maxWorkers, hammer.Config{
```

Comment:
Maybe a `TODO: expose more of these options as flags` ?

Reply:
Exposed